src/tactic/ui/tools/pipeline_canvas_wdg.py

```diff
@@ -7302,6 +7304,20 @@ def get_onload_js(self):
                 node.setStyle("opacity", "0.5");
             }
             spt.pipeline.set_color(node, color);
+
+
+            spt.update.add( node, {
+                search_key: search_keys[process],
+                interval: 3,
```

Comment:
interval = 3 means 3 x 5 seconds = 15 seconds.
Remove interval so that it defaults to 5 seconds,